src/components/ShaclForm/FormRenderer.vue

```diff
@@ -170,6 +171,11 @@ export default class FormRenderer extends Vue {
     return this.isList(field) && values > minCount
   }
 
+  canAddValue(field) {
+    const maxCount: number = _.get(field, 'maxCount', 0)
+    return maxCount === 0 || this.data[field.path].length < maxCount
```

Comment:
Max count 0 has special meaning (not that it would make sense otherwise)? Just wondering, otherwise it could be just the second part after `||`

Reply:
>Max count 0 has special meaning (not that it would make sense otherwise)?

@MarekSuchanek That's a good question. In this case, `maxCount 0` implies that `maxCount` is not defined in the schema, assuming that `maxCount 0` would never occur in a real schema. 

I assumed this because it aligns with the existing behavior of the client for `maxCount 0`.

Actually I'm not sure if this is a valid assumption, as there may be use cases where it is necessary to explicitly forbid the use of a property by setting `maxCount 0`...

>Just wondering, otherwise it could be just the second part after ||

I think we would at least need to check if `maxCount` is defined.

Reply:
@MarekSuchanek my implementation above actually gives rise to #202 because I forgot that lodash `get()` returns the default only when the attribute is `undefined`, but not if it is `null`.

So probably we should just set the default to `null` and test for `null`:

```ts
canAddValue(field) {
    const maxCount: number = _.get(field, 'maxCount', null)
    return maxCount === null || this.data[field.path].length < maxCount
}
```

In addition, I wonder why `isList()` is defined the way it is (`field.maxCount !== 1`). Shouldn't it be something like the following?

```ts
 function isList(field: FormField): boolean { 
   return field.maxCount === null || field.maxCount > 1 
 } 
```

Reply:
Agree, that sounds more error prone if `null` means infinity (any count) 👍🏻

Reply:
Thanks @MarekSuchanek . I verified the expected `isList()` behavior with Luiz and opened #205 accordingly.